### PR TITLE
Refactor to Single Plugin and Add Carpenter Action

### PR DESCRIPTION
**Pull request by mfleader**
## Changes introduced with this PR

Refactor to single plugin and add carpenter github action.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by sandrobonazzola — APPROVED** [review]
LGTM

**Review by ghost — APPROVED** [review]
Looks ok, but I would prefer to move the splitting the repo into at least a separate commit.

File: python/.gitignore [hunk 1]
```diff
@@ -1,162 +0,0 @@
-#From https://raw.githubusercontent.com/github/gitignore/main/Python.gitignore
```
Comment (dustinblack) [inline python/.gitignore:1]: Do you want to move the .gitignore to the repo root instead of deleting it?

**Review by dustinblack — CHANGES_REQUESTED** [review]

File: python/.gitignore (hunk 1 above)
Comment (mfleader) [reply python/.gitignore:1]: i'll copy it over

File: .github/workflows/carpenter.yaml [hunk 2]
```diff
@@ -9,6 +9,9 @@ env:
   IMAGE_TAG: 'latest'
   GITHUB_USERNAME: ${{ github.actor }}
   GITHUB_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+  QUAY_NAMESPACE: ${{ secrets.QUAY_NAMESPACE }}
```
Comment (ghost) [inline .github/workflows/carpenter.yaml:12]: I've added this secret.

**Review by sandrobonazzola — APPROVED** [review]

**Review by dustinblack — APPROVED** [review]

**Pull request merged**